api/src/utils/csv-utils/csv-config-utils.ts

```diff
@@ -118,16 +118,42 @@ export class CSVConfigUtils<StaticHeaderType extends Uppercase<string>> {
     return difference(this.worksheetHeaders, this.worksheetAliasedStaticHeaders);
   }
 
+  /**
+   * Set a static header config. Injects the header config into the CSV static headers config.
+   *
+   * @param {StaticHeaderType} header - The header name
+   * @param {CSVHeaderConfig} headerConfig - The header config
+   * @returns {void}
+   */
+  setStaticHeaderConfig(header: StaticHeaderType, headerConfig: CSVHeaderConfig): void {
+    this._config.staticHeadersConfig[header] = { ...this._config.staticHeadersConfig[header], ...headerConfig };
+  }
+
+  /**
+   * Get the final CSV config
+   *
+   * @returns {CSVConfig<StaticHeaderType>} - The CSV config
+   */
+  getConfig(): CSVConfig<StaticHeaderType> {
+    for (const header of this.configStaticHeaders) {
+      if (!this._config.staticHeadersConfig[header].validateCell) {
+        throw new Error(`Invalid CSV config. Missing 'validateCell' for static header: ${header}`);
+      }
+    }
+
+    return this._config;
+  }
+
   /**
    * Get the cell value from a CSV row.
    *
    * @param {StaticHeaderType} header - The header name
    * @param {CSVRow} row - The CSV row
-   * @returns {unknown} - The cell value
+   * @returns {any} - The cell value
    */
-  getCellValue(header: StaticHeaderType, row: CSVRow) {
+  getCellValue(header: StaticHeaderType, row: CSVRow): any {
```

Comment:
Is this really `any` or can it be like `string | undefined`?

Possibly we might need to check if `row[header] / row[alias]` exists, and then cast it to string or else return undefined?

Reply:
But, ignore if that doesn't help anything or makes things downstream worse

Reply:
Updated